M2/LICENSING-NOTES

```diff
@@ -1,5 +1,24 @@
-See INSTALL file for our license.  Here we document briefly the licenses on the various
-libraries.
+                                   Macaulay2
+                   by Daniel R. Grayson <dan@math.uiuc.edu>
+                and Michael E. Stillman <mike@math.cornell.edu>
+
+                     available from http://Macaulay2.com/
+
+We thank the United States National Science Foundation for generous funding
+to develop Macaulay2.
+
+Macaulay2 is copyright 1993-2016 by Daniel R. Grayson and Michael E. Stillman.
```

Comment:
I suppose this should be 1993-2024?  The date is also in `M2/Macaulay2/README.in`, so I'm not sure how many times we want to duplicate it.

Reply:
Should I just delete it then?

Reply:
I think `LICENSING-NOTES` is maybe a more natural place to put the date, so maybe leave it here to keep updated and remove it from the README?